src/rpc.rs

```diff
@@ -274,8 +274,13 @@ impl State {
         self.accounts.insert(key, data, commitment)
     }
 
-    fn subscription_active(&self, key: Pubkey) -> bool {
-        self.pubsub.subscription_active(key)
+    fn websocket_connected(&self, key: Pubkey) -> bool {
+        self.pubsub.websocket_connected(key)
+    }
+
+    // owner is the subscription for program, if given account belongs to one
```

Comment:
What is this comment about? I don't see any owner around

Reply:
Sorry, slipped through when I was cherry picking from other branch